### ISSUE TITLE
svelte-check: make output machine-readable

### ISSUE THREAD

**Issue by rondonjon** [sveltejs/language-tools#124]
**Is your feature request related to a problem? Please describe.**

The current output of `svelte-check` is human-friendly (e.g. colors, source code snippets) and verbose but at the same time very machine-unfriendly. There is no specification of the output format in the README / manpage / `--help`, and the included source code extracts could be challenging for a parser.

**Describe the solution you'd like**

Please make the output more configurable through CLI parameters, e.g.:

 * a `--ci` parameter for: less verbosity, no colors, and a clearly specified line format, e.g.:
```
    FILE <path>
    <ERROR|WARNING> <line>:<column> "<description (no linebreaks, quotes escaped with backslash)>"
   ...
```
 * a `--verbose` flag for the *current* behaviour

**Describe alternatives you've considered**

In comparison, parsing the current output seems very circumstantial.

**Comment by rondonjon** [comment]
Find pull request [here](https://github.com/sveltejs/language-tools/pull/126)

**Comment by orta** [comment]
Fixed in #126
